test/precompiled/precompiled_test.h

```diff
@@ -2,7 +2,5 @@
 
 #include "precompiled/precompiled.h"
 
-#include "test/test_common/printers.h"
```

Comment:
Things that are in precompiled are depended upon by the envoy_cc_library rule, so it isn't possible to have them built using this rule. printers.h actually depends on files such as common/buffer/buffer_impl.h, which is basically arbitrary implementation files. 

Who uses the pretty printers? Is this GTest? I couldn't find any references to them in the tree.